tests/tests_integration/test_api/test_assets.py

```diff
@@ -86,6 +86,15 @@ def test_list(self, post_spy):
         assert 20 == len(res)
         assert 2 == COGNITE_CLIENT.assets._post.call_count
 
+    def test_partitioned_list(self, post_spy):
+        # stop race conditions by cutting off max created time
+        maxtime = int(time.time() - 3600) * 1000
+        res_flat = COGNITE_CLIENT.assets.list(limit=None, created_time={"max": maxtime})
+        res_part = COGNITE_CLIENT.assets.list(partitions=8, limit=None, created_time={"max": maxtime})
+        assert len(res_flat) > 0
+        assert len(res_flat) == len(res_part)
+        assert [a.id for a in res_flat].sort() == [a.id for a in res_part].sort()
```

Comment:
can also do `assert {a.id for a in res_flat} == {a.id for a in res_part}`

Reply:
right you are :)